doc/source/user_guide/make_container.rst

```diff
@@ -0,0 +1,106 @@
+.. _ref_make_container:
+
+Create your Fluent Docker container
+===================================
+
+.. warning:: You need a valid Ansys license and an Ansys account to
+   follow the steps in this section.
+
+The following procure for creating a Fluent Docker container uses a local
+Ubuntu machine to copy the needed files from the Fluent
+
+
+Requirements
+============
+
+* A Linux machine, preferably with Ubuntu 18.04 or later.
+  CentOS Linux distribution is not supported anymore.
+  This machine needs to have `Docker <https://www.docker.com>`_ installed.
+
+* A valid Ansys account. Your Ansys reseller should have
+  provide you with one.
+
+* The following provided files:
+  
+  * `Dockerfile <https://github.com/ansys/pyfluent/blob/main/docker/fluent/Dockerfile>`_
+  * `copy_docker_files.py <https://github.com/ansys/pyfluent/blob/main/docker/fluent/copy_docker_files.py>`_
+
+
+Procedure
+=========
+
+If you have cloned `PyFluent <https://github.com/ansys/pyfluent>`_ locally then change current working directory to
+`Docker files <https://github.com/ansys/pyfluent/blob/main/docker/fluent>`_ before executing these commands
+otherwise copy these files in an empty folder and execute these commands from that folder.
+
+Specify the Ansys installation directory
+----------------------------------------
+
+Specify the pre-installed Ansys directory as a command line argument.
+
+.. code:: python
+
+    python copy_docker_files.py <path to 'ansys_inc' directory>
```

Comment:
This is running the script as in the next step. What did you intend to write here?

Reply:
Merged 1st step into 2nd. Done. Thank you.